onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -323,6 +323,55 @@ OnnxRuntimeTestSession::OnnxRuntimeTestSession(Ort::Env& env, std::random_device
 #endif
   } else if (provider_name == onnxruntime::kNnapiExecutionProvider) {
 #ifdef USE_NNAPI
+    uint32_t nnapi_flags = 0;
+#ifdef _MSC_VER
+    std::string ov_string = ToMBString(performance_test_config.run_config.ep_runtime_config_string);
+#else
```

Comment:
#ifdef _MSC_VER is unnecessary, NNAPI EP will always be complied by NDK

Reply:
Yeah, It makes sense.